src/.vscode/launch.json

```diff
@@ -18,9 +18,9 @@
             "preLaunchTask": "BuildAndPublishMac", // Replace by BuildAndPublishWindows as needed
             // If you have changed target frameworks, make sure to update the program path.
             // Replace osx-x64 with win-x64 if you are running in windows
-            "program": "${workspaceFolder}/dsc/bin/Debug/netcoreapp3.1/osx-x64/dsc.dll",
-            "args": ["connect", "--service", "mi-webapp-service", "--local-port", "3000", "--namespace", "mi-webapp2", "--use-kubernetes-service-environment-variables"],
-            //"args": ["prep-connect", "--service", "stats-api"],// "--local-port", "3001", ],
+            "program": "${workspaceFolder}/dsc/bin/Debug/net6.0/win-x64/dsc.dll",
```

Comment:
preLaunchTask has BuildAnPublishMac, but path here has win, maybe update it to osx for consistency with the prelaunch task?